src/stream/src/executor/sync_kv_log_store.rs

```diff
@@ -579,20 +660,31 @@ impl<S: StateStore> SyncedKvLogStoreExecutor<S> {
 
 struct SyncedLogStoreBuffer {
     buffer: VecDeque<(u64, LogStoreBufferItem)>,
+    current_size: usize,
     max_size: usize,
     next_chunk_id: ChunkId,
     metrics: KvLogStoreMetrics,
+    flushed_count: usize,
 }
 
 impl SyncedLogStoreBuffer {
+    /// Returns true if there are flushed items in the buffer.
```

Comment:
To make the method name more consistent with the implementation, may just be `self.flushed_count`? We can check `self.is_empty()` in where we wanted to check `self.current_size > 0`.

Reply:
Fixed